File: rest/connect-apps/list-get-example-1/list-get-example-1.6.x.java
// Install the Java helper library from twilio.com/docs/java/install
import com.twilio.sdk.TwilioRestClient;
import com.twilio.sdk.TwilioRestException;
import com.twilio.sdk.resource.instance.ConnectApp;
import com.twilio.sdk.resource.list.ConnectAppList;

public class Example { 

  // Find your Account Sid and Token at twilio.com/user/account
  public static final String ACCOUNT_SID = "ACXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX";
  public static final String AUTH_TOKEN = "your_auth_token";

  public static void main(String[] args) throws TwilioRestException {
    TwilioRestClient client = new TwilioRestClient(ACCOUNT_SID, AUTH_TOKEN);

    
    
    ConnectAppList connectApps = client.getAccount().getConnectApps();
    
    // Loop over connectApps and print out a property for each one.
    for (ConnectApp connectApp : connectApps) {
      System.out.println(connectApp.getFriendlyName());
    }
    
  }
}

File: rest/short-codes/list-get-example-1/list-get-example-1.6.x.py
# Download the Python helper library from twilio.com/docs/python/install
from twilio.rest import Client

# Your Account Sid and Auth Token from twilio.com/user/account
account_sid = "ACXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX"
auth_token = "your_auth_token"
client = Client(account_sid, auth_token)

# A list of short_code objects with the properties described above
for short_code in client.short_codes.list():
    print(short_code.short_code)
